app/util/AWS.scala

```diff
@@ -34,6 +34,8 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val kinesisStreamName = config.getString("aws.kinesis.streamName").get
 
+  lazy val kinesisReindexStreamName = config.getString("aws.kinesis.reindexStreamName").get
```

Comment:
Yes of course, I just stuck this comment here as it was when I was reading this that it popped into my head :)